lib/screens/home.dart

```diff
@@ -24,27 +24,40 @@ class _HomePageState extends State<HomePage> {
   final firebaseFlag = false;
   final scaffoldKey = new GlobalKey<ScaffoldState>();
   List gfComponents = [
-    {'icon': CupertinoIcons.heart_solid, 'title': 'PPG', 'route': PPG()},
+    {
+      'icon': CupertinoIcons.heart_solid,
+      'title': 'PPG',
+      'route': PPG(),
+      'enabled': false
+    },
     {
       'icon': Icons.video_call,
       'title': 'Telehealth',
       'authentication': "firebase",
       'failRoute': LoginFirebaseScreen(),
-      'route': Telehealth()
+      'route': Telehealth(),
+      'enabled': true,
     },
     {
       'icon': Icons.people,
       'title': 'Patient List',
       'route': PatientListPage(),
       'authentication': "webapp",
-      'failRoute': LoginScreen()
+      'failRoute': LoginScreen(),
+      'enabled': true,
```

Comment:
Let's make this parameter optional, that way we don't need to manage it for every card.
`disabled: true`